primary/src/block_synchronizer/mod.rs

```diff
@@ -896,8 +896,10 @@ impl BlockSynchronizer {
                 .expect("Worker id not found")
                 .name;
 
```

Comment:
Put the debug! log here and no need to clone the batch_ids and primary_peer_name any more.

Reply:
I kept it in the same location but made it reference the field in `message` so that we could stil log "Sent" instead of "Sending"

`primary_peer_name` still needs the clone because it's a function argument that gets reused multiple times in the loop